--- 4/wiringOP/wiringPi/OrangePi.h      2019-07-04 23:26:24.567801691 +0000
+++ 3/wiringOP/wiringPi/OrangePi.h      2019-07-04 15:28:09.840601127 +0000
@@ -52,6 +52,17 @@
 #define MAP_SIZE_L                         (4096 * 2)
 #endif

+/************** OrangePi Zero ***********************/
+#if CONFIG_ORANGEPI_H3 || CONFIG_ORANGEPI_ZERO
+#define GPIOA_BASE                         (0x01C20000)
+#define GPIO_NUM                           (0x40)
+#define GPIO_BASE_MAP                      (0x01C20800)
+#define MEM_INFO                           (1024)
+#define GPIOL_BASE                         (0x01F02c00)
+#define GPIOL_BASE_MAP                     (0x01F02000)
+#define MAP_SIZE_L                         (4096 * 2)
+#define GPIO_PWM_OP                                               (0x01C21000)
+#endif
 /************** OrangePi H3 ***********************/
 #if CONFIG_ORANGEPI_H3 || CONFIG_ORANGEPI_H3_ZEROPLUS2
 #define GPIOA_BASE                         (0x01C20000)